src/components/AstronautsPage/components/AstronautsList/components/AstronautsItem/components/AstronautsItemImage/AstronautsItemImage.tsx

```diff
@@ -1,13 +1,20 @@
 import React from 'react';
 import { CardMedia } from '@mui/material';
-import { astronautsData } from '../../../../../../mocks/astronautsData';
 import './AstronautsItemImage.scss';
 
-export const AstronautsItemImage = () => (
-  <CardMedia
-    className="astronauts-item-image"
-    component="img"
-    image={astronautsData.image}
-    alt="astronaut"
-  />
+interface AstronautsItemImage {
+  image: string;
+}
+
+export const AstronautsItemImage = React.memo(
+  ({ image }: AstronautsItemImage) => (
+    <CardMedia
+      className="astronauts-item-image"
+      component="img"
+      image={image}
+      alt="astronaut"
+    />
+  ),
 );
+
+AstronautsItemImage.displayName = 'AstronautsItemImage';
```

Comment:
Why do you need it here?

Reply:
It was addition to React.Memo, i removed it